os/board/rtl8721csm/src/component/os/tizenrt/rtk_blemgr.c

```diff
@@ -143,6 +144,8 @@ trble_result_e trble_netmgr_init(struct bledev *dev, trble_client_init_config *c
 		ret = rtw_ble_server_init(server);
 	}
 #endif
+	/* Set BLE(44):WiFi(56) time slot */
+	(void)rltk_coex_set_ble_scan_duty(56);

```

Comment:
How about making defines like below?
```
#define BLE_TIME_SLOT 44
#define WIFI_TIME_SLOT 56
```
Then it is easy to understand the code.

Reply:
Ok, I will fix it